src/main/java/dev/joopie/jambot/repository/SpotifyToYoutubeFinder.java

```diff
@@ -0,0 +1,22 @@
+package dev.joopie.jambot.repository;
+
+import dev.joopie.jambot.models.SpotifyToYoutube;
+import dev.joopie.jambot.repository.base.BaseFinder;
+import io.ebean.DB;
+
+public class SpotifyToYoutubeFinder extends BaseFinder<SpotifyToYoutube> {
```

Comment:
TODO; final class with private constructor to prevent init. We want this via our repository class